docs/Developers/Git_Basics.rst

```diff
@@ -84,11 +84,33 @@ command:
   **myfork** is just a name we used for simplicity. You can
   name it however you want.
 
+The next thing you should know before getting down to work is how to check on
+the changes you've made to your project, and your current branch. The
+ability to check your current branch is also extremely important, as you'll see
+in the next section. The command to check this information is:
+
+::
+
+    $ git status
+
+Before we move onto the next section, you need to know about a very important
+branch called master. Master is the default branch that git checkouts for you
+when you clone a repository. It's our policy here at coala to never develop
+on your master branch. This is why we create new branches, which leads us to the
+next section.
+
 Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
-will work.
+will work. Do not change files when you are on your master branch. If you
+submit a Pull Request from your master branch, maintainers will assume that you
+didn't read this guide. coala developers may even reject your work
+(even if it is a good patch), because you are showing you haven't checked our
+documentation. The reason why you should never develop on your master
+branch is because your master branch should always be synchronized with the
```

Comment:
`your master branch` -> `your fork's master branch`
Might be good to add `fork` to be more explicit.